lib/contentTypeParser.js

```diff
@@ -205,21 +205,21 @@ function rawBody (request, reply, options, parser, done) {
 }
 
 function getDefaultJsonParser (onProtoPoisoning, onConstructorPoisoning) {
-  return defaultJsonParser
+  return (req, body, done) => defaultJsonParser(req, body, done, { onProtoPoisoning, onConstructorPoisoning })
```

Comment:
This adds one more object allocation where it was not happening before. I would prefer we avoid it as it was in the previous version of thje code.